pyproject.toml

```diff
@@ -3,7 +3,14 @@
 requires = [
     "setuptools",
     "wheel",
-    "Cython>=0.28.5",
-    "numpy>=1.13.3",
-    "scipy>=0.19.1",
+    "Cython>=0.29.13", 
+    "scipy>=1.2.2",
```

Comment:
Let's put here > minimal supported scipy version (cf readme) for now. I'm not actually sure what's the right thing to do. Asked a question on the mailing list: https://mail.python.org/pipermail/scipy-dev/2020-April/024113.html

Reply:
Yeah, will contact the mailing list;)

Reply:
> Yeah, will contact the mailing list;)

I mean that we just need to wait for the reply there. But I think we can merge this as is, and make a follow up PR if needed later.